src/main/com/aca/homework/week10/entry/EntryImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.aca.homework.week10.entry;
+
+public class EntryImpl<K, V> implements Entry<K, V> {
+
+    private K K;
```

Comment:
the variable name should start with lowercase and should be descriptive, readable name